insta-comments-inserter/main/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"github.com/codeuniversity/smag-mvp/insta-comments-inserter"
+	"github.com/codeuniversity/smag-mvp/kafka"
+	"github.com/codeuniversity/smag-mvp/service"
+	"github.com/codeuniversity/smag-mvp/utils"
+)
+
+func main() {
+	postgresHost := utils.GetStringFromEnvWithDefault("POSTGRES_HOST", "127.0.0.1")
+	postgresPassword := utils.GetStringFromEnvWithDefault("POSTGRES_PASSWORD", "")
+
+	qReaderConfig, qWriterConfig, _ := kafka.GetInserterConfig()
```

Comment:
`qWriterConfig` will be `nil` if `isUserDiscovery` is false 
`GetInserterConfig` is not generalised  well for this case, feel free to just get the env vars without the util func